stratum/lib/security/credentials_manager.cc

```diff
@@ -113,7 +113,6 @@ ::util::Status CredentialsManager::Initialize(bool secure_only) {
               kFileRefreshIntervalSeconds);
       auto tls_opts = std::make_shared<TlsChannelCredentialsOptions>();
       tls_opts->set_certificate_provider(certificate_provider);
```

Comment:
Two things:
- First, you need to provide more information in the commit comment about the compilation issues you are resolving and why this is the best way of addressing them. The reviewer needs this information in order to evaluate the correctness of your change. At Google, failure to provide an adequate commit comment is grounds for peremptory rejection of a changelist. See [Writing good CL descriptions](https://google.github.io/eng-practices/review/developer/cl-descriptions.html) and [Chapter 9](https://learning.oreilly.com/library/view/software-engineering-at/9781492082781/ch09.html) of _Software Engineering at Google_ for more information.
- Silently removing a line of code from a shared file that we propose to upstream is setting a landmine in the code. It is essential that you leave breadcrumbs for the developer who will be upstreaming it. Rather than deleting the line, you should disable it with `#if 0 / #endif` and add comments between the `#if 0` and the line you're disabling saying _why_ you are disabling the line and providing any additional information that will be of use to other custodians of the code. In particular, _what are the side effects of removing this line?_ You are, after all, telling the code not to do something that it did previously.

It looks as if this change disables verification of the certificate. If so, it has security consequences! You're secretly removing security protections from the code! This is not a good thing.

Reply:
Yes, I'm aware that you provided some additional information as a comment on the PR. This helps the reviewer, but it doesn't leave information in the git archive (i.e., it's not part of the commit comment) and it doesn't leave any trace in the code.

Reply:
I have updated the code and pushed an update with details of why we are disabling. Wrapped it around a `#if 0` flag

Reply:
As an added measure, I could make a call to the new API `set_verify_server_certs()` and set it to `true`, even though gRPC documentation says that the default is true.

However, this might break if others are using older version of gRPC since the API did not exist in the past.

Leaving it out since default is true.